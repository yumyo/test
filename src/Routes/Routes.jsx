import React from 'react';
import { BrowserRouter, Switch, Route } from 'react-router-dom';

import Nav from '../Layouts/Nav';
import Review from '../Pages/Review';
import Task from '../Pages/Task';

const Routes = () => (
    <BrowserRouter>
        <Nav>
            <Switch>
                <Route exact path="/review" component={Review} />
                <Route exact path="/task" component={Task} />
            </Switch>
        </Nav>
    </BrowserRouter>
);

export default Routes;
