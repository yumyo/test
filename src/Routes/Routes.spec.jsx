import React from 'react';
import { shallow } from 'enzyme';
import Routes from './Routes.jsx';

describe('<Routes />', () => {
    let routes;

    beforeEach(() => {
        routes = shallow(<Routes />);
    });

    it('renders', () => {
        expect(routes).toMatchSnapshot();
    });
});
