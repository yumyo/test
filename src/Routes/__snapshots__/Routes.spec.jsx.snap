// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Routes /> renders 1`] = `
<BrowserRouter>
  <Nav>
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/review"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/task"
      />
    </Switch>
  </Nav>
</BrowserRouter>
`;
